lib/cnet/punt/punt_kernel.c

```diff
@@ -37,40 +38,47 @@
 #include <cnet_eth.h>
 #include <net/cne_udp.h>
 #include <hexdump.h>
+#include <cnet_tcp.h>
 
 #include <cnet_node_names.h>
 #include "punt_kernel_priv.h"
-#include "tun_alloc.h"
 
 #define PREFETCH_CNT 6
 
 static __cne_always_inline void
 punt_kernel_process_mbuf(struct cne_node *node, pktmbuf_t **mbufs, uint16_t cnt)
 {
     punt_kernel_node_ctx_t *ctx = (punt_kernel_node_ctx_t *)node->ctx;
-    struct iovec v[cnt];
-    int fd;
 
-    if ((fd = tun_get_fd(ctx->tinfo)) >= 0) {
+    if (ctx->sock >= 0) {
+        struct cne_ipv4_hdr *ip4;
```

Comment:
we can add ipv6 later

Reply:
We can create a punt6_kernel.c  version later, this way we do not need to worry about combining them into one.